CHANGELOG.md

```diff
@@ -1,17 +1,24 @@
-## v1.0.2
-### 2024-04-05
+## v1.1.0 - 2024-04-30
+
+### Changed
+Changes the computation for an output image's default scale extent. Previously
+we considered ICD preferred ScaleExtents as covering the entire globe or pole.
+This change now takes just the input image bounds and converts them to the target crs
+and uses that transformed boundry as the default region to make a scale extent from.
```

Comment:
It might be worth consolidating descriptions of the behaviour for scale extents, scale sizes, heights, width, tiling, etc, in a section in `README.md`, probably right near the start of the file. (Or at least a minor tweak to update the [example usage notebook](https://github.com/nasa/harmony-browse-image-generator/blob/main/docs/HyBIG-Example-Usage.ipynb))

I'm mainly thinking that as more folks consider using HyBIG for their browse image generation, we need to be able to point at a cheat-sheet of the expected behaviour.

Reply:
I agree, and so did David in this comment https://nsidc.slack.com/archives/CCGM48T8B/p1712233204288769 How about I write that ticket and we prioritize it rather than adding it to this one?

Reply:
I could add a tweak to the doc notebook, I kind of completely forgot that existed.  You have a suggestion?

Reply:
bugs.earthdata.nasa.gov/browse/DAS-2166